### PR TITLE
Clarify name uniqueness tradeoff

### PR DESCRIPTION
**Pull request by maelle**
@eliocamp would this help?

### CODE REVIEW

**Comment by eliocamp** [comment]
Yes, much clearer!

**Comment by maelle** [comment]
Thanks @eliocamp!

@yabellini could you please review the (automatic) Spanish translation? Thank you!

File: pkg_building.es.Rmd [hunk 1]
```diff
@@ -18,7 +18,9 @@ Para leer por qué vale la pena enviar un paquete a rOpenSci siguiendo las recom
     Puedes comprobar si el nombre de tu paquete está disponible, es informativo y no es ofensivo utilizando el [paquete `available`](https://github.com/ropenscilabs/available).
     En particular, *no* elijas un nombre de paquete que no esté disponible en CRAN o en Bioconductor.
 
--   Un nombre poco común tiene la ventaja de que su uso es más fácil de detectar (habrá menos falsos positivos al buscar el nombre en la búsqueda de código de GitHub para que tú o rOpenSci evalúen su uso) y de buscar (para que quienes lo usen puedan buscar "cómo usar el paquete <nombre_paquete>?" en internet).
+- Existe un equilibrio entre las ventajas de un nombre de paquete único y un nombre de paquete menos original. 
+    - Un nombre de paquete más único podría ser más fácil de rastrear (para ti y para nosotros para evaluar el uso del paquete, por ejemplo, menos falsos positivos al escribir su nombre en la búsqueda de código de GitHub) y de buscar (para los usuarios que preguntan "cómo usar el paquete bla" en un motor de búsqueda).
```
Comment (yabellini) [inline pkg_building.es.Rmd:22]:
```suggestion
    - Un nombre de paquete más único, o poco común, tiene la ventaja de que su uso es más fácil de detectar (habrá menos falsos positivos al buscar el nombre en la búsqueda de código de GitHub) y también es más sencillo de buscar (para que quienes lo usan puedan buscar "cómo usar el paquete <nombre_paquete>" en internet). 
```

**Comment by maelle** [comment]
@eliocamp what do you mean?

**Comment by eliocamp** [comment]
That I used this version in reviewing the Spanish translation, so this PR shouldn't require a new automatic translation and review.

**Comment by maelle** [comment]
@eliocamp oh ok I think the branch is out of date then, I'll open a new PR and use the English text. sorry @yabellini

**Pull request closed without merging**